File: Labs/Week7/Ant/args.java
// args.java - Looking at command-line args
// Demonstrates:	args []
//
// 3/04 
//
//

import java.io.*;

public class args
{
	public static void main( String[] args )
	{
		int l = args.length;
		for( int i = 0; i< l; ++i )
			System.out.println( args[i] );
		System.out.println( "\nOr, try this:\n" );
		for( String S : args )
			System.out.println( S );
	}
} 	// class args

File: Labs/Week4/Python/args.py
#!/usr/bin/python
#
# args.py - reading command-line args
#
# Kurt Schmidt
# 5/06
#
#	Demonstrates: sys.argv
#

import sys

argc = len( sys.argv )

print "There are", argc, "arguments"

for i in range( argc ) :
	print i, sys.argv[i]

print ""

print "--OR--"

for a in sys.argv :
	print a

